runtimes/web/src/apu-worklet.ts

```diff
@@ -161,7 +167,7 @@ class APUProcessor extends AudioWorkletProcessor {
         }
     }
 
-    process (inputs, [[ output ]], parameters) {
+    process (inputs: any, [[ output ]]: number[][][], parameters: any) {
```

Comment:
Could we perhaps use `unknown` rather than `any` here? `unknown` prevents accidental non-typed usage while `any` does not. (this applies to any other usages of `any` too)